packages/features/settings/outOfOffice/CreateOrEditOutOfOfficeModal.tsx

```diff
@@ -118,9 +147,43 @@ export const CreateOrEditOutOfOfficeEntryModal = ({
           <div className="h-full px-1">
             <DialogHeader
               title={
-                currentlyEditingOutOfOfficeEntry ? t("edit_an_out_of_office") : t("create_an_out_of_office")
+                currentlyEditingOutOfOfficeEntry
+                  ? t("edit_an_out_of_office")
+                  : oooType === "team"
+                  ? t("create_ooo_dialog_team_title")
+                  : t("create_an_out_of_office")
               }
             />
+
+            {/* In case of Team, Select Member for whom OOO is created */}
+            {oooType === "team" && (
+              <div className="mb-4 mt-4 h-16">
```

Comment:
Can we make sure the spaces are all the same. There is not enough space between label and dropdown and too much space under dropdown: 
![Screenshot 2024-09-20 at 2 17 08 PM](https://github.com/user-attachments/assets/485a1f08-ca09-4bf0-82b6-2e206bd2ff1d)

Reply:
updated , It was like that for Redirect TeamMember selection before and somehow overlooked this one, updated there as well.
<img width="509" alt="Screenshot 2024-09-21 at 2 34 41 PM" src="https://github.com/user-attachments/assets/950b1063-fe48-44db-a98c-293df8bcad49">